hudi-cli/src/main/java/org/apache/hudi/cli/commands/CompactionCommand.java

```diff
@@ -195,9 +196,10 @@ public String scheduleCompact(
     HoodieTableMetaClient client = checkAndGetMetaClient();
     boolean initialized = HoodieCLI.initConf();
     HoodieCLI.initFS(initialized);
-
+    HoodieTimeGeneratorConfig timeGeneratorConfig = HoodieTimeGeneratorConfig.newBuilder()
+        .withPath(HoodieCLI.basePath).build();
     // First get a compaction instant time and pass it to spark launcher for scheduling compaction
-    String compactionInstantTime = HoodieActiveTimeline.createNewInstantTime();
+    String compactionInstantTime = HoodieActiveTimeline.createNewInstantTime(timeGeneratorConfig, HoodieCLI.conf);
 
```

Comment:
Kind of think the invoke should be:

`TimeGenerators.getInstance(HoodieCLI.basePath, HoodieCLI.conf).createNewInstantTime`

Reply:
Fire the ticket to track this refactor: [HUDI-6860](https://issues.apache.org/jira/browse/HUDI-6860)